typescript-express-reviews/README.md

```diff
@@ -0,0 +1,32 @@
+# typescript-express-reviews
+
+This sample demonstrates integrating Mongoose with [`express`](https://www.npmjs.com/package/express).
+
+## Setup
+
+Make sure you have a local stargate instance running as described on the [main page](../README.md) of this repo.
```

Comment:
also set up environment variables?

Reply:
yes I am hitting this as well.. Can we have meaningful defaults for these, cause we anyway depend on that startup script that brings jsonapi docker compose up:

```
STARGATE_JSON_API_URL=http://localhost:8080/v1/my_keyspace
STARGATE_JSON_AUTH_URL=http://localhost:8081/v1/auth 
STARGATE_JSON_USERNAME=cassandra 
STARGATE_JSON_PASSWORD=cassandra
```

Reply:
ah, I see the `.env` examples now, maybe just say this should be the first step `cp .env.example .env`?

Reply:
Fixed by bd016dc28e7b5f1f3088c86bd474edb9ae8dfd33